bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties

```diff
@@ -35,46 +35,19 @@ thing-type.config.tr064.fritzbox.rejectedCallDays.label = Rejected Call Days
 thing-type.config.tr064.fritzbox.wanBlockIPs.description = List of IPs that can be blocked for WAN access.
 thing-type.config.tr064.generic.host.label = Host
 thing-type.config.tr064.generic.host.description = Host name or IP address.
 thing-type.config.tr064.generic.password.label = Password
 thing-type.config.tr064.generic.refresh.label = Refresh Interval
-thing-type.config.tr064.generic.timeout.label = Timeout
-thing-type.config.tr064.generic.timeout.description = Timeout for all requests (SOAP requests, phone book retrieval, call lists, ...).
 thing-type.config.tr064.generic.user.label = Username
 thing-type.config.tr064.subdevice.refresh.label = Refresh Interval
 thing-type.config.tr064.subdevice.uuid.label = UUID
 thing-type.config.tr064.subdevice.uuid.description = UUID of the sub-device
 thing-type.config.tr064.subdeviceLan.macOnline.label = MAC Online
-thing-type.config.tr064.subdeviceLan.macOnline.description = List of MACs for "online" status detection (format: 11:11:11:11:11:11).
+thing-type.config.tr064.subdeviceLan.macOnline.description = List of MACs for "online" status detection (format: 11:11:11:11:11:11). One MAC-Address per row.
 thing-type.config.tr064.subdeviceLan.refresh.label = Refresh Interval
 thing-type.config.tr064.subdeviceLan.uuid.label = UUID
 thing-type.config.tr064.subdeviceLan.uuid.description = UUID of the sub-device
-
-# thing types config
-
-profile-type.transform.PHONEBOOK.label = Phone Book
-profile.config.transform.PHONEBOOK.phonebook.label = Phone Book
-profile.config.transform.PHONEBOOK.phonebook.description = The phone book name.
-profile.config.transform.PHONEBOOK.matchCount.label = Match Count
-profile.config.transform.PHONEBOOK.matchCount.description = The number of matching numbers required for matches. Matching is done from the far end of numbers. The default value is 0, which is considered as "match everything".
-profile.config.transform.PHONEBOOK.phoneNumberIndex.label = Phone Number Index
-profile.config.transform.PHONEBOOK.phoneNumberIndex.description = The index of the phone number that is resolved from a CallItem-State (StringListType), 0 (default) or 1.
-
-# actions
-
-phonebookLookupActionLabel = lookup a phone number
-phonebookLookupActionDescription = Lookup a phone number.
-phonebookLookupActionOutputLabel = Name
-phonebookLookupActionOutputDescription = The name associated with the phone number (or the phone number if no match can be found).
-phonebookLookupActionInputPhoneNumberLabel = Phone Number
-phonebookLookupActionInputPhoneNumberDescription = The phone number to be resolved.
-phonebookLookupActionInputPhoneBookLabel = Phone Book
-phonebookLookupActionInputPhoneBookDescription = The name of the the phone book.
-phonebookLookupActionInputMatchesLabel = Match Count
-phonebookLookupActionInputMatchesDescription = The number of digits matching the incoming value, counted from far right (default is 0 = all matching). Negative numbers skip digits from the left.
```

Comment:
These are still referenced by `@text/` references here:
https://github.com/openhab/openhab-addons/blob/be6b799ca447b55ce69a068e65af57d3d843d11b/bundles/org.openhab.binding.tr064/src/main/java/org/openhab/binding/tr064/internal/phonebook/PhonebookActions.java#L42-L66

Reply:
I’ll have a look, probably revert everything as it was. 
This is something the i18n maven command does not pickup?

Reply:
The tool only picks strings from the XML sources. For references like above, the I18N properties file **is** the source.

Reply:
Checked and fixed the actions. Thanks for the details. Still learning alot here ;-)

Reply:
I could not find any reference for the `profile-type.transform.xxx` but don't know where to expect those so i might be looking at the wrond place.

Reply:
They have a suspicious duplicated header "# thing types config", so probably they are not relevant anymore. If you want to be 100% sure, you could check git history and find the commit where they moved, and check other changes in that commit.

Reply:
Seems like they are merged by https://github.com/openhab/openhab-addons/commit/f9695f2d8dd234bdba9b96a5c902b37798b9fded#diff-6114fe980cebbb1ca878651701ea434d443f2ed2071402e103f41683d5679349
and somehow referenced from [bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/config/phonebookProfile.xml](https://github.com/openhab/openhab-addons/commit/f9695f2d8dd234bdba9b96a5c902b37798b9fded#diff-6114fe980cebbb1ca878651701ea434d443f2ed2071402e103f41683d5679349)

The uri in the xml is profile:transform:PHONEBOOK and in the translation is profile.config.transform.PHONEBOOK 

i'm to unfamiliair with this piece of translation / binding specifics, it also goes beyond the goal of this PR. So do you agree that i revert the removal of these lines and you merge it as is?